datasets.md

```diff
@@ -1,72 +1,31 @@
-| SiSEC real-world noise                            | public spaces                        | N/S                            | read                               | 0.3                | 43500               | no         | 2010        | free                         | to add |
-| SiSEC dynamic                                     | cocktail party                       | N/S                            | read                               | 0.2                | 43500               | no         | 2010 - 2011 | free                         | to add |
-| CHiME 1, CHiME 2 Grid                             | domestic                             | UK English                     | command                            | 70                 | 2                   | no         | 2011 - 2012 | free                         | to add |
-| CHiME 2 WSJ0                                      | domestic                             | US English                     | read                               | 78                 | 2                   | no         | 2012        | free given WSJ0 (1.5 k$)     | to add |
-| ETAPE                                             | TV/radio debates, outdoor interviews | French                         | spontaneous                        | 42                 | 1                   | 1          | 2012        | ?                            | to add |
-| GALE                                              | TV dialog                            | Mandarin, Arabic               | spontaneous                        | 120 - 251 per lang | 1                   | no         | 2013        | 3.5 - 7 k$ per lang          | to add |
-| REVERB SimData                                    | domestic, office                     | UK English                     | read                               | 25                 | 8                   | no         | 2013        | free given WSJCAM0 (1.75 k$) | to add |
-| Sheffield Wargames Corpus                         | cocktail party                       | UK English                     | spontaneous                        | 7                  | 92                  | 3          | 2013        | free                         | to add |
-| DIRHA                                             | domestic                             | various                        | command, read, spontaneous         | 11                 | 40                  | no         | 2014        | free (partial avail.)        | to add |
-| CHiME 3                                           | public spaces                        | US English                     | read                               | 48                 | 6                   | no         | 2015        | free given WSJ0 (1.5 k$)     | to add |      |
+## Submitting a new dataset
 
+To submit a new dataset, please use the following form:
 
+<iframe src="https://script.google.com/macros/s/AKfycbzQcBYOyjDJh3U3HijedXClUfLgajavmlNDOQXgvjhiqWqdR5MRuEg7a8UtVjOEqu86/exec" 
```

Comment:
Great to have the form to add new entries.
Is it possible to give pre-defined categories for Use-case, speaking style etc
We could also add others and then the user needs to input manually. 
I don't know if it is easy or possible to do.
Use-case: Noise reduction, Speech separation, Noise-robust ASR, Meeting ASR, Speaker recognition, Diarization 
speaking style: Command, Read speech, Spontaneous, conversation

The above is just an idea of potential categories, but there may be more or more adequate ones.

Reply:
Yes, this should be doable. I'll add it.

Reply:
I updated the form to use Google Forms instead, which has a much nicer interface. You can see the form here: https://forms.gle/PBQ5ipjqTEsDwhYZ9